### PR TITLE
Fix golint on importer pkg

### PR DESCRIPTION
**Pull request by j-griffith**
Fix golint errors on importer

### CODE REVIEW

File: pkg/importer/dataStream.go [hunk 1]
```diff
@@ -37,59 +38,59 @@ type DataStreamInterface interface {
...
+	"s3":    rdrS3,
+	"tar":   rdrTar,
+	"xz":    rdrXz,
 }
 
-// Return a dataStream object after validating the endpoint and constructing the reader/closer chain.
+// NewDataStream returns a DataStream object after validating the endpoint and constructing the reader/closer chain.
 // Note: the caller must close the `Readers` in reverse order. See Close().
-func NewDataStream(endpt, accKey, secKey string) (*dataStream, error) {
+func NewDataStream(endpt, accKey, secKey string) (*DataStream, error) {
```
Comment (mhenriks) [inline pkg/importer/dataStream.go]: Nice.  This always bugged me.

File: pkg/importer/dataStream.go [hunk 2]
```diff
@@ -37,59 +38,59 @@ type DataStreamInterface interface {
...
 	if len(accKey) == 0 || len(secKey) == 0 {
-		glog.V(Vadmin).Infof("%s and/or %s are empty\n", IMPORTER_ACCESS_KEY_ID, IMPORTER_SECRET_KEY)
+		glog.V(common.Vadmin).Infof("%s and/or %s are empty\n", common.IMPORTER_ACCESS_KEY_ID, common.IMPORTER_SECRET_KEY)
```
Comment (mhenriks) [inline pkg/importer/dataStream.go]: This is maybe the subject of a new issue/pr but doesn't kubernetes code just use the number for the `glog.V` functions?  Personally, I prefer `glog.V(3).Infof` to `glog.V(common.Vadmin).Infof`.  But I am happy to be gone with the dot import for `common`.

**Comment by jeffvance** [comment]
Needs rebase

File: pkg/importer/dataStream.go [hunk 3]
```diff
@@ -37,59 +38,59 @@ type DataStreamInterface interface {
...
 
 const (
-	RdrHttp = iota
-	RdrS3
-	RdrFile
-	RdrGz
-	RdrMulti
-	RdrTar
-	RdrXz
+	rdrHTTP = iota
```
Comment (jeffvance) [inline pkg/importer/dataStream.go]: are there functional tests that use these constants?

File: pkg/importer/dataStream.go (hunk 2 above)
Comment (j-griffith) [reply pkg/importer/dataStream.go]: @mhenriks to stage things a bit I split that change out:  https://github.com/kubevirt/containerized-data-importer/pull/408

Saving common for last as it's the most invasive change, will wait for the simple lint changes to merge then get that posted with hopefully minimal confusion.  Thanks!

File: pkg/importer/dataStream.go [hunk 4]
```diff
@@ -412,7 +413,7 @@ func (d *dataStream) isoSize() (int64, error) {
 	const (
 		vdTypeOff       = 0
 		typeLen         = 1
-		vdIdOff         = 1
+		vdIDOff         = 1
```
[the remark this answers is not in the document]
Comment (j-griffith) [reply pkg/importer/dataStream.go]: Yeah, it's just a camel-case thing. Golint requires all the things be camel case, and common abbreviations like ID UUID etc it's *smart* enough to pick them out and say "hey, these should be all the same case" (whether it's LC or UC) depending on how it falls in the name.

File: pkg/importer/dataStream.go [hunk 5]
```diff
@@ -300,23 +301,23 @@ func (d *dataStream) appendReader(rType int, x interface{}) {
...
-func (d dataStream) topReader() io.ReadCloser {
-	return d.Readers[len(d.Readers)-1].Rdr
+func (d DataStream) topReader() io.ReadCloser {
+	return d.Readers[len(d.Readers)-1].rdr
 }
 
 // Based on the passed in header, append the format-specific reader to the readers stack,
 // and update the receiver Size field. Note: a bool is set in the receiver for qcow2 files.
-func (d *dataStream) fileFormatSelector(hdr *image.Header) (err error) {
+func (d *DataStream) fileFormatSelector(hdr *image.Header) (err error) {
```
[the remark this answers is not in the document]
Comment (jeffvance) [reply pkg/importer/dataStream.go]: Thanks for the explanation. I have more to learn about golint...

**Comment by jeffvance** [comment]
lgtm

**Pull request merged**